mindsdb_native/libs/controllers/transaction.py

```diff
@@ -325,7 +325,11 @@ def run(self):
         self._call_phase_module(module_name='ModelInterface', mode='predict')
 
         if self.lmd['return_raw_predictions']:
-            return self.hmd['predictions']
+            self.output_data = PredictTransactionOutputData(
+                transaction=self,
+                data=self.hmd['predictions']
+            )
+            return self.output_data
```

Comment:
Hmh, if need be we can do it like this for now, but I think it's non-ideal, since `PredictTransactionOutputData` adds the kind of overhead we *shouldn't* have as part of the quick methods.